src/uu/rm/src/rm.rs

```diff
@@ -306,7 +307,18 @@ fn remove_dir(path: &Path, options: &Options) -> bool {
                 }
             }
             Err(e) => {
-                show_error!("removing '{}': {}", path.display(), e);
+                match e.raw_os_error() {
+                    Some(39) => {
```

Comment:
why do you do this? it cannot be generalized ?

Reply:
I apologize, can you explain what you mean? I made it a match statement so that errors could be made for other error codes. How would you like me to do it instead?